pytorch_lightning/loggers/base.py

```diff
@@ -297,6 +297,20 @@ def log_graph(self, model: "pl.LightningModule", input_array=None) -> None:
         """
         pass
 
+    def log_text(self, *args, **kwargs) -> None:
```

Comment:
is it misleading for users to need to figure out which loggers actually implement this vs which don't? the expectation for those users will be that `NotWandBlogger.log_text` actually logs text instead of the no-op here

Reply:
Not sure, some of the other methods are not implemented in all loggers either.
We could instead add a warning that appears once if the method is not implemented in one of the loggers used.

Reply:
I agree with @ananthsub . Since users will call these methods explicitly, we should also raise an Error if not implemented

Reply:
Gread idea! I'm now raising `NotImplementedError`